src/write_events.py

```diff
@@ -0,0 +1,40 @@
+import pandas as pd
+from ata_db_models.models import Event
+from sqlalchemy.dialects.postgresql import insert
+from sqlalchemy.orm import sessionmaker
+
+from src.helpers.logging import logging
+
+logger = logging.getLogger(__name__)
+
+
+def write_events(df: pd.DataFrame, Session: sessionmaker) -> int:
```

Comment:
🔧 I know sqlalchemy's doc examples use this upper-case `Session`, but I'd prefer if we keep our code pythonic and use a lower-case variable here. Since it is a `sessionmaker`, I'd suggest calling it `session_factory` or `session_maker` or something. Irritates me that the class `sessionmaker` is not capitalized either, darn SQLAlchemy folks!
```suggestion
def write_events(df: pd.DataFrame, session_factory: sessionmaker) -> int:
```

Reply:
Totally, yeah!

Reply:
Done!